### ISSUE TITLE
Email: Intopia Feedback - WCAG 3.0 FPWD (4 - Testing)

### ISSUE THREAD

**Issue by jspellman** [w3c/wcag3#281]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0015.html): 

FPWD: We would like constructive feedback on the testing approach, and examples of why you would or would not implement it in your organization.

Significant work would be required to adopt the testing approach within our organisation, but we will consider adoption as the guidelines continue to develop.

Initial areas that would impact on adopting the testing approach include:

  *   Clarity on recording test results – if results need to be recorded per atomic test result for every element, this will increase the amount of work needed to prepare documentation. The greatest impact will be for clients that have retesting that needs to be completed (as multiple sets of documentation may be required dependent on testing methodology)
  *   The suitability of atomic tests will need to be reviewed given that no assistive technology is required as a part of the tests
  *   For any tests that are technology specific, clarity on how often these will be reviewed (and superseded) is needed
  *   List(s) of tests should be collated as the guidelines are finalised – the current content structure makes it difficult for testers to work through a seamless flow of tests

**Comment by hidde** [comment]
The issues described in Intopia's email are related to the first public working draft of WCAG 3 and are no longer relevant to the [latest draft that was published 10 September](https://www.w3.org/TR/wcag-3.0/). As issues have either been addressed or are no longer relevant, I am closing this issue.
